shipment_advice/models/res_config_settings.py

```diff
@@ -10,3 +10,7 @@ class ResConfigSettings(models.TransientModel):
     shipment_advice_outgoing_backorder_policy = fields.Selection(
         related="company_id.shipment_advice_outgoing_backorder_policy", readonly=False
     )
+    shipment_advice_auto_validate = fields.Boolean(
+        string="Enable Shipment Advice Auto Validation",
+        config_parameter='shipment_advice.shipment_advice_auto_validate',
+    )
```

Comment:
This should be a related field to the company. 
```suggestion
    shipment_advice_auto_validate = fields.Boolean(
        related="company_id.shipment_advice_auto_validate",
    )
```

Reply:
readonly=False